go.mod

```diff
@@ -8,10 +8,17 @@ require (
 	github.com/mattn/go-runewidth v0.0.14
 	github.com/mitchellh/colorstring v0.0.0-20190213212951-d06e56a500db
 	github.com/olekukonko/tablewriter v0.0.5
+	github.com/shirou/gopsutil v3.21.11+incompatible
```

Comment:
used to check if the llama.cpp server process was able to start on the given port